Wuunder/vendor/nategood/httpful/README.md

```diff
@@ -1,8 +1,8 @@
 # Httpful
```

Comment:
Should this file/folder be included?

Reply:
removed, .gitignore was updated after commit